modules/logging-enabling-loki-alerts.adoc

```diff
@@ -1,30 +1,37 @@
 // Module included in the following assemblies:
-// logging-5-7-configuration
+//
+// logging/logging_alerts/log-storage-alerts.adoc
 
 :_content-type: PROCEDURE
-[id="logging-loki-alerts_{context}"]
-= Enabling log based alerts with Loki
-Loki alerting rules use link:https://grafana.com/docs/loki/latest/logql/[LogQL] and follow link:https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/#recording-rules[Prometheus formatting]. You can set log based alerts by creating an `AlertingRule` custom resource (CR). `AlertingRule` CRs may be created for `application`, `audit`, or `infrastructure` tenants.
+[id="logging-enabling-loki-alerts_{context}"]
+= Enabling log-based alerts with Loki
+
+LokiStack log-based alerts are triggered by providing link:https://grafana.com/docs/loki/latest/query/[LogQL] expressions to the LokiStack `ruler` component. The Loki Operator manages a ruler that is optimized for the selected LokiStack size, which can be 1x.extra-small, 1x.small, or 1x.medium.
+
+[NOTE]
+====
+1x.extra-small is for demo purposes only, and is not supported.
```

Comment:
This will change with 5.8 when `extra-small` becomes a new supported size and `demo` is the new size for demo purposes (probably not relevant for this PR yet, though).

Reply:
Let's cover that as part of https://issues.redhat.com/browse/OBSDOCS-478 maybe? I had asked Peri about this but I think never got a response, maybe this was what was meant by that Jira?